portal-kernel/src/com/liferay/portal/kernel/service/persistence/impl/BasePersistenceImpl.java

```diff
@@ -1038,6 +1038,9 @@ protected T updateImpl(T model) {
 	@Deprecated
 	protected boolean finderCacheEnabled = true;
 
+	protected int valueObjectFinderCacheListThreshold = GetterUtil.getInteger(
```

Comment:
Hi @dantewang ,
After a second thought, I do not think we can add this here.
The reason is, a release version may use a updated service builder which contains the logic you added, but they may not have this backport to BasePersistenceImpl.
So let's move this to the concrete PersistenceImpl

Reply:
@tinatian OK, got it.